ucm2/alcatel-idol3/HiFi.conf

```diff
@@ -53,6 +53,43 @@ SectionDevice."Earpiece" {
 	}
 }
 
+SectionDevice."Headphones" {
+	Comment "Headphones playback"
+
+	EnableSequence [
+		cset "name='RX1 MIX1 INP1' RX1"
+		cset "name='RX2 MIX1 INP1' RX2"
+		cset "name='RDAC2 MUX' RX2"
+		cset "name='HPHL' 1"
+		cset "name='HPHR' 1"
+		## gain to  0dB
+		cset "name='RX1 Digital Volume' 128"
+		## gain to  0dB
+		cset "name='RX2 Digital Volume' 128"
```

Comment:
I don't understand why you're still enabling the PM8916 headphone output. Should drop all this :)